docs/document/website/versioned_docs/version-1.4.0/getting-started/quick-start.md

```diff
@@ -0,0 +1,195 @@
+---
+id: version-1.4.0-quick-start
+title: Quick Start
+sidebar_label: Quick Start
+original_id: quick-start
+---
+
+## What is Kintone UI Component?
+
+Kintone UI Component is a library that enables Kintone developers to build forms with Kintone's styling with ease.
+
+### Example for Desktop components
+
+![desktop components](assets/desktop_components.png)
+
+### Example for Mobile components
+
+![mobile components](assets/mobile_components.png)
+
+## Preparation: Create App
+
+In this tutorial, you will place a Button component inside the header menu space on the Record List page of a Kintone app.
+
+First, follow the below steps to create an app to install Kintone UI Component.
+
+1. Create a blank app. ([Creating an App from Scratch](https://get.kintone.help/k/en/user/create_app/tutorial.html))<br>
+You do not need to set up any fields in this app.
+
+## Installation
+
+Kintone UI Component supports installation via a UMD, a CDN, or a npm package.<br>
+You can choose to import or implement the library according to the environment such as the browser environment or Node.js environment.<br>
+In this article, we will show you how to install and implement using each approach.
+
+> See the Components section in the sidebar for more details on each component.<br>
+> (Additional components will be added in order.)
+
+### Use the UMD
+> We recommend that you should upgrade Kintone UI Component to the latest version for your projects. <br>
+> From v1.4.0, please use `Kucs["1.x.x"]` instead of `Kuc` and specify your expected version (ex. `Kucs["1.4.0"]`).<br>
+>The rendered components' tags and class names will include the version number.<br>
+> When using a version before v1.4.0, please use `Kuc` as a global variable but **note that it may be conflicting when adding two or more `kuc.min.js` files** on Kintone app plug-in or customization.
```

Comment:
```suggestion
> When using a version before v1.4.0, please use `Kuc` as a global variable but **note that it may be conflicting when adding two or more `kuc.min.js` files** on Kintone app plug-in or customization.<br>
```